### ISSUE TITLE
Memcached Changes the behavior at update failure.

### ISSUE THREAD

**Issue by SawamiWataru** [personium/personium-core#35]
### Personium source code
MemcachedClient#put()
```
if (this.spyClient.replace(key, expiresIn, object).get()) {
    return true;
} else {
    return this.spyClient.add(key, expiresIn, object).get();
}
```
If false is returned, an error occurs.

### Problem
Problems may occur at the time of simultaneous access.

Assume that API to be executed concurrently is [API-1] [API-2].
A phenomenon occurs when the following processing order is reached.

1. [API-1] Call replace.
2. [API-1] False is returned because there is no information in memcache.
3. [API-2] Call replace.
4. [API-2] False is returned because there is no information in memcache.
5. [API-1] Call add.
6. [API-1] Write information to memcache and return true.
7. [API-2] Call add.
8. [API-2] False is returned because there is information in memcache.

As a result, it is not an abnormal state, but it is an error.

**Comment by SawamiWataru** [comment]
### Solution
If add fails, respond by calling replace again.

**Comment by SawamiWataru** [comment]
Solved in v1.5.4.
